opte/src/engine/ip4.rs

```diff
@@ -171,7 +171,7 @@ impl MatchExact<Protocol> for Protocol {
 }
 
 #[derive(
-    Clone, Debug, Deserialize, Eq, Ord, PartialEq, PartialOrd, Serialize,
+    Clone, Debug, Deserialize, Eq, Ord, PartialEq, PartialOrd, Serialize, Copy,
```

Comment:
At some point I forgot Copy is really just about move semantics and got my wires crossed to associate it with bcopy'ing, even when looking at a reference -- which is totally not how this works at all, haha. You are right, no reason these types shouldn't be Copy.

Reply:
Ah, ok that makes sense. Yeah, it's just indicating that moving out of a value means the original can still be referenced, since it's implicitly copied. I'll put the derive in order now, and then mark these and the places resolved. Thanks!